lib/services/database.dart

```diff
@@ -59,7 +59,11 @@ class DatabaseService {
     doc.updateData({"infoGiocatori": FieldValue.arrayUnion(val)});
   }
 
-  Stream<List<InfoGiocatore>> getTeamMembersInfo(String idSquadra) {
+  Future<List<InfoGiocatore>> getTeamMembersInfoList(String idSquadra) async {
```

Comment:
Cambierei il nome in `getTeamMembersInfoFuture` per analogia con `getTeamMembersInfoStream`